docs/01-overview/02-main-areas/service-management/smgt-02-brokers-overview.md

```diff
@@ -0,0 +1,130 @@
+---
+title: Service Brokers
+---
+
```

Comment:
I'd suggest to add sub-headlines in the topic to utilize the right-hand navigation and split the document in smaller sections